src/execution_plan/optimizations/optimizer.c

```diff
@@ -21,9 +21,16 @@ void optimizePlan(ExecutionPlan *plan, AST *ast) {
     /* Try to reduce a number of filters into a single filter op. */
     reduceFilters(plan);
 
+    /* Remove redundant SCAN operations. */
+    // reduceScans(plan);
```

Comment:
Delete commented lines; this optimization is performed above.

Reply:
👍